recipes/openmm/conda_build_config.yaml

```diff
@@ -0,0 +1,6 @@
+cuda_compiler:
+- nvcc
+cuda_compiler_version:
+- 10.0
+channel_targets:
+- conda-forge testing
```

Comment:
What is this for?

Reply:
We don't want packages in `conda-forge/main` without manually testing them with an actual GPU. Given the current OpenMM [docs](http://docs.openmm.org/7.4.0/userguide/application.html#installing-openmm), `conda-forge` has precedence over `omnia` (where `openmm` resides now), and any new users that follow those instructions would obtain an untested (potentially broken) package.

Reply:
Why does `conda-forge` have precedence over `omnia`? Would figure it would be the other way around?

Reply:
A while back (a couple years ago now) when we were planning all this out, we wanted to eventually move every package over from `omnia` to `conda-forge`, so us `omnia` devs could eventually stop having to maintain a separate conda-based build system ourselves. The hard part of that was going to be resolving all the dependencies, which we had to maintain/copy a large number of them over to the omnia channel. 

To prepare for this migration, we decided that `conda-forge` would be the de-facto dependency source, and we only maintained on `omnia` the minimum set of packages we had to. This basically meant that the packages which were were all OpenMM dependent at some level. The goal was, once OpenMM came to `conda-forge`, it would be very easy for all the other packages to migrate over as well since all they would have to do (in theory) was create a `staged-recipe` and the dependencies would basically already be resolved since we were relying on `conda-forge` as the higher priority channel anyways.